app/src/main/java/ch/hikemate/app/ui/components/HikeCard.kt

```diff
@@ -68,13 +95,15 @@ fun HikeCard(
           Row(
               verticalAlignment = Alignment.CenterVertically,
               modifier = Modifier.fillMaxWidth().height(IntrinsicSize.Min)) {
-                Box(
+                ElevationGraph(
+                    elevationData = elevationData,
                     modifier =
                         Modifier.weight(1f)
                             .fillMaxHeight()
                             .background(
                                 MaterialTheme.colorScheme.primary.copy(alpha = 0.1f),
-                                RoundedCornerShape(4.dp)))
+                                RoundedCornerShape(4.dp)),
```

Comment:
Warning: IMO, this should be in the `ElevationGraph` composable. This is global and should never change

Reply:
Maybe not, we might want to change the appearance of the ElevationGraph (including the background) if we show it on other locations